### ISSUE TITLE
Figure out how to remove deleted pages from global search results

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#3524]
### Summary

**What we are after:**
Figure out how to remove deleted pages from global search results.  

### Related issues

_List any relevant related issue(s)_
- #3376 
- #3523 

### Completion criteria

- [x] Figure out how to delete these three pages from global search results: See [comment](https://github.com/fecgov/fec-cms/issues/3524#issuecomment-609791495).
- https://www.fec.gov/help-candidates-and-committees/keeping-pac-records-nonconnected/misappropriated-nonconnected-pac-funds/ (THIS PAGE WAS DELETED ) (89)
- https://www.fec.gov/help-candidates-and-committees/making-disbursements-pac/notices-required-nonconnected-pac-solicitations-nonconnected-pac/ (THIS PAGE WAS DELETED) (20)
- **[Fixed]** https://www.fec.gov/help-candidates-and-committees/candidate-taking-receipts/contribution-limits-candidates/ (6077) - See #3376 and #3523

**Comment by jason-upchurch** [comment]
emailed search.gov to remove urls from index.

**Comment by jason-upchurch** [comment]
@dorothyyeager @johnnyporkchops @pkfec After talking with John, I sent an email to search.gov, and they responded today that the searches no longer show in the results. I verified myself before sending this note, but wanted to see if one of you would be willing to double check me before I close this. Thanks!

**Comment by dorothyyeager** [comment]
Hi @jason-upchurch - thank you for working on this. Sorry I missed your tagging me untl now! Unfortunately, I'm still seeing the pages in the search results that shouldn't be there...

Search term: "misappropriated funds"
![image](https://user-images.githubusercontent.com/24437369/78562633-b40b1f80-77e7-11ea-86aa-c13a7ba12f37.png)

Search term: "PAC solicitations"
![image](https://user-images.githubusercontent.com/24437369/78562723-d1d88480-77e7-11ea-9e27-c9e0e773d934.png)

Search term: "Contribution limits" Problem is that the old retired page is the top suggested result with a star:
![image](https://user-images.githubusercontent.com/24437369/78562925-267bff80-77e8-11ea-8ddf-395e4115af93.png)

All of these pages have working redirects, but would like to get them out of the search altogether. Let me know if you have any questions.

**Comment by jason-upchurch** [comment]
Thank you @dorothyyeager I'm looking into this!

**Comment by jason-upchurch** [comment]
@dorothyyeager cc @patphongs I believe both issues are now resolved. I did some testing on my end, but can you verify it looks good when you have a moment? Thank you!

**Comment by dorothyyeager** [comment]
Hi Jason. Contribution limits page issues and misappropriated funds issues are fixed.

Still getting the wrong page for "PAC solicitations": URL returned should be: https://www.fec.gov/help-candidates-and-committees/making-disbursements-pac/notices-required-nonconnected-solicitations/ - it's confusing; hope they didn't remove the wrong URL from the results.

**Comment by jason-upchurch** [comment]
Hi @dorothyyeager, when I picked this ticket up, I only dealt with these two URLs:
* https://www.fec.gov/help-candidates-and-committees/keeping-pac-records-nonconnected/misappropriated-nonconnected-pac-funds/ 

* https://www.fec.gov/help-candidates-and-committees/candidate-taking-receipts/contribution-limits-candidates/ 

I'm not familiar with the one above, and I don't see that one in our `i14y` drawer (cc @patphongs).

Since this issue dealt with removing incorrect URLs that were showing up in search results, and that work is done, do you have any concerns with us opening a new "bug" issue for the URL you are seeing with info indicating what IS showing up and what SHOULD BE showing up?

We can keep it on this issue too, but I'm worried it's getting a bit muddled.

**Comment by dorothyyeager** [comment]
Huh, it's listed above in https://github.com/fecgov/fec-cms/issues/3524#issuecomment-609791495 but I did forget to highlight it in the screencap. 

Basically the issue is that when you search the term "PAC solicitation" you get an outdated page instead of the newer page it was redirected to.

Old deleted URL to take out of results: https://www.fec.gov/help-candidates-and-committees/making-disbursements-pac/notices-required-nonconnected-pac-solicitations-nonconnected-pac/ 

URL that should be in results: https://www.fec.gov/help-candidates-and-committees/making-disbursements-pac/notices-required-nonconnected-solicitations/

I'm happy to open a new ticket if that's helpful so we can close this one out.

**Comment by jason-upchurch** [comment]
Will revisit once April indexing is done to verify that https://www.fec.gov/help-candidates-and-committees/making-disbursements-pac/notices-required-nonconnected-solicitations/ appears in the `json` from `scrape_cms_pages`

**Comment by JonellaCulmer** [comment]
Adding the April indexing ticket (#3608)  as a blocker for this work. 
cc @pkfec

**Comment by jason-upchurch** [comment]
> Adding the April indexing ticket (#3608) as a blocker for this work.
> cc @pkfec

Good call, @JonellaCulmer , thank you!

**Comment by pkfec** [comment]
@jason-upchurch : I have scraped the latest wagtail pages and indexed search.gov site on 04.24.2020 and i closed the ticket here :https://github.com/fecgov/fec-cms/issues/3608.

**Comment by jason-upchurch** [comment]
Thank you @pkfec !! I'll remove the blocked from this issue since it's not really appropriate anymore.

**Comment by jason-upchurch** [comment]
@pkfec @JonellaCulmer I will just check to see if it is removed from our search and report back here so reviews can be completed.

**Comment by jason-upchurch** [comment]
@dorothyyeager thanks for your help (and patience) testing this out. I just went through the highlighted examples you gave (https://github.com/fecgov/fec-cms/issues/3524#issuecomment-609791495) and it looks like the search issues are resolved. Are you able to verify when you have time? Thank you!

**Comment by dorothyyeager** [comment]
Thanks @jason-upchurch for hanging in there on this! Happy to say the issue is resolved and we can close this ticket.

**Comment by jason-upchurch** [comment]
Thank you for your help @dorothyyeager! Closing ticket now
